src/api/daemon.go

```diff
@@ -33,9 +34,78 @@ func getSystemdDaemon(daemonName string) (Daemon, error) {
 		Name:      daemonName,
 		IsActive:  isActive,
 		IsEnabled: isEnabled,
+		InitLevel: "", // TODO
 	}, nil
 }
 
+func getOpenRCDaemon(ctx context.Context, daemonName string) (Daemon, error) {
+	cmd := exec.CommandContext(ctx, "rc-service", daemonName, "status")
```

Comment:
Do you remember how sql injection works?
For me this code looks like great way to make "command injection"

Reply:
How would you implement it then?

Reply:
I would check if the name contains any invalid name characters like e.g.
whitespace characters, `;`, etc. (I'm not sure but I guess `;` is an illegal daemon name character)